src/main/scala/cromwell/util/PathUtil.scala

```diff
@@ -1,11 +1,23 @@
 package cromwell.util
 
+import java.nio.file.{Path, Paths}
+
+import cromwell.engine.io.gcs.GcsFileSystem
+
 
 object PathUtil {
```

Comment:
Could this be renamed to something like `PathString` and moved to the engine package object?